spec/rubocop/cop/rspec/repeated_example_group_body_spec.rb

```diff
@@ -226,6 +226,50 @@
     RUBY
   end
 
+  it 'skips `skip` and `pending` statements with arguments' do
+    expect_no_offenses(<<-RUBY)
+      context 'processed' do
+        skip 'some reason'
+      end
+
+      context 'processed' do
+        skip 'some reason'
+      end
```

Comment:
Now when I look at it, it seems that there's something wrong, and it's the repeated context title.
Do you mind to make them different?
The example with `describe '#load' do` isn't the best reference, it's a miss.

Reply:
I'll try update them to clarify use case.